addons/mrp/models/mrp_workcenter.py

```diff
@@ -592,18 +585,24 @@ def _default_time_stop(self):
         workcenter_id = self.workcenter_id.id or self.env.context.get('default_workcenter_id')
         return self.env['mrp.workcenter'].browse(workcenter_id).time_stop if workcenter_id else 0.0
 
-    workcenter_id = fields.Many2one('mrp.workcenter', string='Work Center', required=True, index=True)
-    product_id = fields.Many2one('product.product', string='Product', required=True)
-    product_uom_id = fields.Many2one('uom.uom', string='Product Unit', related='product_id.uom_id')
-    capacity = fields.Float('Capacity', default=1.0, help="Number of pieces that can be produced in parallel for this product.")
+    workcenter_id = fields.Many2one('mrp.workcenter', string='Work Center', required=True)
+    product_id = fields.Many2one('product.product', string='Product')
+    product_uom_id = fields.Many2one('uom.uom', string='Unit', default=lambda self: self.env.ref('uom.product_uom_unit'),
```

Comment:
since product_id is not required this one is probably not required neither?

Reply:
in specs : allow to define capacities cross products depending on UoM
( the one from BoM )